bootstrapsavagexr.sh

```diff
@@ -0,0 +1,290 @@
+#!/bin/bash
```

Comment:
Very unlikely user will run it from terminal to see the output. How often does the binary change? If not much then it would be better to use extra-data.

Reply:
Moved the downloading of the Savage XR installer to extra-data action, but I can't actually run the installer from apply_extra script as its a graphical installer, which is a shame.

Either way its simplified the bootstrap script.

Reply:
Removed hash bang, not sure what else you want here so some guidance would be appreciated.

Reply:
Hide terminal when starting application from desktop icon.